test/data/zimfiles/create_test_zimfiles

```diff
@@ -74,7 +74,8 @@ make__poor__zim()
     sed -e 's!I/image.png!http://a.io/pic.png!' main.html > external_link.html
     sed -e 's/article1/non_existent/' main.html > dangling_link.html
     sed -e 's!A/article1!../../oops!' main.html > outofbounds_link.html
-    sed -e '6 a <meta http-equiv=refresh content=0; url=file://${(pwd)}/redirect_loop.html/>' main.html > redirect_loop.html
+    command_set="sed -e '6 a <meta http-equiv=refresh content=\"0;URL=redirect_loop.html\">' main.html"
+    eval $command_set > redirect_loop.html
```

Comment:
Yes, it does pass. 
Furthermore, if I change that particular line to 
`command_set="sed -e '6 a <meta http-equiv=refresh content=\"0;URL=main.html\">' main.html"`
(i.e. the redirect changes to main.html and is therefore a normal redirect instead of a loop)
and regenerate the `poor.zim` file and re-run the tests, it _does not_ pass.

Reply:
And whenever I regenerate the `poor.zim` using `./create_test_zimfiles poor  `
I get the output:
`Resolve redirect`
`set index`
`poor.zim was successfully created`

The first line of the output suggests to me that the redirect is being dealt with. 
(I may be wrong)

Reply:
I used the `<meta` tag so as to introduce the http redirect.

Reply:
Ooops! I completely missed the point of HTTP redirects. For some reason I kept thinking about HTML links (that I have most dealt with in zimcheck).

Reply:
So there are no conceptual issues with your PR. Go on with adding a command line option/flag for redirect loop detection. Also add a couple of test cases for that command line flag akin to, for examle, `zimcheck.favicon_goodzimfile` and `zimcheck.favicon_poorzimfile`.